src/RNTuple/Writing/TFileWriter.jl

```diff
@@ -505,30 +531,33 @@ function write_rntuple(file::IO, table; file_name="test_ntuple_minimal.root", rn
     tdirectory32_obs = rnt_write_observe(file, Stubs.tdirectory32)
     dummy_padding2_obs = rnt_write_observe(file, Stubs.dummy_padding2)
 
-    RBlob1_obs = rnt_write_observe(file, Stubs.RBlob1)
-    rntAnchor_update[:fSeekHeader] = UInt32(position(file))
-    rnt_header = UnROOT.RNTupleHeader(zero(UInt64), rntuple_name, "", "ROOT v6.33.01", [
-    UnROOT.FieldRecord(zero(UInt32), zero(UInt32), zero(UInt32), zero(UInt16), zero(UInt16), 0, -1, -1, string(only(input_schema.names)), RNTUPLE_WRITE_TYPE_CPPNAME_DICT[input_T], "", ""),
-    ], [UnROOT.ColumnRecord(RNTUPLE_WRITE_TYPE_IDX_DICT[input_T]..., zero(UInt32), 0x00, 0x00, 0),], UnROOT.AliasRecord[], UnROOT.ExtraTypeInfo[])
+    RBlob1 = UnROOT.RBlob(; fNbytes = 0x00DC, fVersion = 0x0004, fObjLen = 0x000000BA, fDatime = Stubs.WRITE_TIME, fKeyLen = 34,
+    fCycle = 0x0001, fSeekKey = position(file), fSeekPdir = 100, fClassName = "RBlob", fName = "", fTitle = "")
+    RBlob1_update = Dict{Symbol, Any}()
+    RBlob1_obs = rnt_write_observe(file, RBlob1)
+    field_records, col_records = schema_to_field_column_records(table)
+    rnt_header = UnROOT.RNTupleHeader(
```

Comment:
I got a bit lost with the dense code lines here, maybe put a little bit of white space and indents 😅

Reply:
yea this part is totally manual right now.

we build each block of things as they appear in the file, and manually construct "observable"-like structure for each of them.

eventually they should be automated once we're more certain what kind of flexibility is needed